lib/jazzy/sourcekitten.rb

```diff
@@ -149,6 +149,9 @@ def self.documented_child?(doc)
     def self.should_document?(doc)
       return false if doc['key.doc.comment'].to_s.include?(':nodoc:')
 
+      # Always document Objective-C declarations.
+      return true if Config.instance.objc_mode
```

Comment:
Because at the moment, jazzy only works in a single language at a time, and lots of things will have to change to support mixed language projects, so until then I'd rather not make any assumptions as to how that future mixed support will work and keep things simpler for what we're currently doing.